src/main/java/com/ericsson/ei/repository/SubscriptionRepository.java

```diff
@@ -11,10 +11,10 @@
 @Component
 public class SubscriptionRepository implements ISubscriptionRepository {
     
-    @Value("${subscription.collection.name}")
+    @Value("${subscriptionCollectionName}")
     private String collectionName;
     
-    @Value("${database.name}")
+    @Value("${subscriptionDataBaseName}")
```

Comment:
database name is for all collections not only for subscription
Camelcase is not used in properties. Standard is point delimited.